charts/midaz/templates/ledger/secrets.yaml

```
@@ -0,0 +1,28 @@
+{{- if and .Values.ledger.enabled (not .Values.ledger.useExistingSecret) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "midaz-ledger.fullname" . }}
+type: Opaque
+data:
+  # =============================================================================
+  # ONBOARDING MODULE SECRETS
+  # =============================================================================
+  DB_ONBOARDING_PASSWORD: {{ .Values.ledger.secrets.DB_ONBOARDING_PASSWORD | default "lerian" | b64enc | quote }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)